README.md

```diff
@@ -3,6 +3,33 @@
 AudioMonster manipulates and transcodes audio.
 It wraps a number of different command line binaries such as sox, lame, flac, twolame, and ffmpeg.
 
+## Dependencies
+
+The following binary tools are required. They are available from most OS package managers. E.e. for OS X use homebrew:
```

Comment:
[stickler mode on]
I think
"managers. E.e for OS X use homebrew:"

should read something like:
"managers, i.e. for OS X use homebrew:"
[stickler mode off]